sass/_a1-home-header.scss

```diff
@@ -0,0 +1,74 @@
+//home header styling
+//this includes the appbox logo in the left corner and the navigation bar in the right corner at the very top of the page.
+.home {
+  color: $primary-font-color;
+  margin: 0;
+}
+.home-h1 {
+  margin-bottom: 1em;
+}
+.a1-home-header {
+  background-color: $background-A;
+  padding-bottom: .5em;
+}
+.home-nav {
+	display: inline;
+  margin-bottom: 1em;
```

Comment:
Watch your spacing -- as a team, you need to decide on spaces or tabs (and how many) and make sure you're all using the same.

Ultimately, the goal here is to make the file feel as if it's been written by one person, regardless of how people contributed to it.